### DISCUSSION TITLE
Add bitquery.to, please

### DISCUSSION THREAD

**Discussion by eis2k** [rotki/rotki#4016] [Archive]
Hey. I must trace mined Ether that was stolen. Therefore I ordered rotki premium which already has some very good features to achieve this. But I was wondering if it's possible to add bitquery to the external services. The money flow abilities are very good but to get it all together bitquery isn't the right place. Also the feature to generate a sankey flow chart to visualize the money flow is fantastic. And last but not least the coin path feature to identify transaction even through hundreds of hobs is more then awesome. 
Imho this would be a great benefit for rotki to add this page as external service. With this rotki would do a big step for AML and fraud detection

**Comment by LefterisJP** [comment]
relevant issue: https://github.com/rotki/rotki/issues/1524
